doc/modules/model_evaluation.rst

```diff
@@ -394,6 +396,42 @@ from the :ref:`example_model_selection_plot_confusion_matrix.py` example):
     documents.
 
 
+Multilabel confusion matrix
+---------------------------
+
+The :func:`multilabel_confusion_matrix` function evaluates
+classification accuracy by computing the `confusion matrix
+<http://en.wikipedia.org/wiki/Confusion_matrix>`_ for the case of multilabel
+classification.
+
+The multilabel confusion matrix is of shape ``(n_labels, 4)``.
+Each row contains the values for the 4 expressions of the F-Score namely
+:math:`TP, FP, TN, FN`.
+
+For each label (row) the values are computed as follows :
```

Comment:
This is described more succinctly under Binary Classification. Reference that instead.